dev/acf-export.json

```
@@ -593,7 +593,56 @@
         "label_placement": "top",
         "instruction_placement": "label",
         "hide_on_screen": "",
-        "active": 1,
+        "active": true,
+        "description": ""
+    },
+    {
+        "key": "group_5c9e1c1417520",
+        "title": "Syndication Fields",
+        "fields": [
+            {
+                "key": "field_5c9e1c1c15df3",
+                "label": "Promote on Main Site",
+                "name": "post_main_site_story",
+                "type": "true_false",
+                "instructions": "When checked, the story will appear in the news feed on the UCF.edu.",
```

We can remove the third "the" in this sentence:
```suggestion
                "instructions": "When checked, the story will appear in the news feed on UCF.edu.",
```